kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -93,7 +97,12 @@ public KafkaProxyInitializer(FilterChainFactory filterChainFactory,
 
     @Override
     public void initChannel(Channel ch) {
-
+        if (!ch.hasAttr(SESSION_ID_ATTRIBUTE_KEY)) {
+            String sessionId = MobyNamesGenerator.getRandomName() + "_" + ch.id().asShortText();
```

Comment:
Using some random name requires adding an extra production dependency. So what's the benefit above just using `ch.id().asShortText()`?

Reply:
To quote the PR description:

>I used the Moby name generator so there was something eye catching for humans to look for in the logs rather than just the channel ID.

I find it considerably easier to follow the logs by eye when they contain words rather than some short code. Short text is is also documented as not being globally unique, and I haven't dug into over what scope it could collide.